utils/storybook/customThemes.ts

```diff
@@ -23,3 +23,30 @@ export const customColorTheme: PartialCanvasTheme = {
     },
   },
 };
+
+export const customColorThemeWithAction: PartialCanvasTheme = {
+  palette: {
+    primary: {
+      main: 'purple',
+      contrast: 'turquoise',
+    },
+    action: {
+      main: 'red',
```

Comment:
Should we consider making this a different color than a color used on the `DeleteButton`?

Reply:
yeah I can update it